micromanager_gui/_core_widgets/_presets_widget.py

```diff
@@ -100,6 +105,7 @@ def _on_cfg_set(self, group: str, preset: str) -> None:
             with signals_blocked(self._combo):
                 self._combo.setCurrentText(preset)
                 self._combo.setStyleSheet("")
+                self._set_cfg_data_as_tool_tip(preset)
```

Comment:
it seems like you're manually calling `_set_cfg_data_as_tool_tip` whenever the text of `self._combo` changes... so perhaps you should just connect to the `currentTextChanged` signal?

```
self._combo.currentTextChanged.connect(self._update_tooltip)

def _update_tooltip(self, preset):
    self._combo.setToolTip(str(self._mmc.getConfigData(self._group, preset)) if preset else '')
```